src/Serilog.Sinks.DiagnosticTrace/project.json

```diff
@@ -0,0 +1,30 @@
+﻿{
+  "version": "2.0.0-beta-*",
+  "description": "The diagnostic trace sink for Serilog - Simple .NET logging with fully-structured events",
```

Comment:
Could drop the extra verbiage and keep it to `"The diagnostic trace sink for Serilog"` or maybe `"The System.Diagnostics.Trace sink for Serilog"`?